src/features/academic-overview/StudentHolds.tsx

```diff
@@ -32,7 +43,15 @@ export const StudentHolds: React.FC = () => {
           <span> {singularPlural(data.length, 'hold')} on your student account.</span>
           {data.length > 0 && (
             <HoldsList>
-              {data.map((h, i) => h.description && <li key={i}>{h.description}</li>)}
+              {data.map(
+                (h, i) =>
+                  h.description && (
+                    <li key={i}>
+                      <HoldTitle>{h.description.toUpperCase()}</HoldTitle>
```

Comment:
The screenshot and the mock data looks like these descriptions are uppercase but we don't necessarily want to force it with a `.toUpperCase()` here..  the data source is inconsistent, I'm sure, but in this case we should just keep it as-is `h.description`.